docs/CONTRIBUTING.md

```diff
@@ -1,11 +1,20 @@
-# Contribution guidelines [WIP]
+# Contribution guidelines 
 
-## How to contribute to project? [Coming soon]
+## How to contribute to project? 
+All external contribution to this project is welcome. To propose changes, we encourage contributors to open a new branch and propose their changes by opening a new PR. 
 
 ## Having problems?
-Have you encountered an error? A critical step in troubleshooting is being able to reproduct the problem. Instructions to publicly reproduce errors using GitHub Actions can be found in our [guide to reproduce errors.](/docs/REPRODUCE_ERRORS.md)
+Have you encountered an error? A critical step in troubleshooting is being able to reproduce the problem. Instructions to publicly reproduce errors using GitHub Actions can be found in our [guide to reproduce errors.](/docs/REPRODUCE_ERRORS.md)
 
-## Code template [Coming soon]
+## Code template
+
+###Issue and PR templates
+Issue templates have been designed to ease the processes to suggest a new feature or report a bug. We encourage contributors not only to format their PR's title following the [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/), but also use the PR templates made available on this repository.
+
+###Coding style
+Sticking to a single consistent and documented coding style for this project is important to ensure that code reviewers dedicate their attention to the functionality of the validation, as opposed to disagreements about the coding style (and avoid bike-shedding). 
```

Comment:
I assume https://en.wikipedia.org/wiki/Law_of_triviality is "bike-shedding" here. Please link to it.